packages/cloud-cognitive/src/components/EditFullPage/EditFullPage.stories.js

```diff
@@ -0,0 +1,452 @@
+/**
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { useState } from 'react';
+import { carbon } from '../../settings';
+import {
+  getStoryTitle,
+  prepareStory,
+} from '../../global/js/utils/story-helper';
+import { action } from '@storybook/addon-actions';
+import { CreateFullPage } from '../CreateFullPage';
+import { CreateFullPageStep } from '../CreateFullPage/CreateFullPageStep';
+import { pkg } from '../../settings';
+
+import { EditFullPage } from '.';
+import mdx from './EditFullPage.mdx';
+
+import styles from '../CreateFullPage/_storybook-styles.scss';
+
+const storyClass = 'create-full-page-stories';
+const blockClass = `${pkg.prefix}--create-full-page`;
+
+import {
+  Checkbox,
+  TextInput,
+  NumberInput,
+  InlineNotification,
+  Toggle,
+  Tooltip,
+  RadioButtonGroup,
+  RadioButton,
+  FormGroup,
+  Row,
+  Column,
+} from 'carbon-components-react';
+
+export default {
+  title: getStoryTitle(EditFullPage.displayName),
+  component: CreateFullPage,
+  subcomponents: { CreateFullPageStep },
+  parameters: {
+    styles,
+    layout: 'fullscreen',
+    docs: { page: mdx },
+    controls: { sort: 'requiredFirst' },
+  },
+  decorators: [
+    (story) => <div className={`${storyClass}__viewport`}>{story()}</div>,
+  ],
+};
+
+const defaultFullPageProps = {
+  nextButtonText: 'Next',
+  backButtonText: 'Back',
+  cancelButtonText: 'Cancel',
+  submitButtonText: 'Update',
```

Comment:
```suggestion
  cancelButtonText: 'Cancel changes',
  submitButtonText: 'Save changes',
```

Reply:
Actually, this could be okay as is. In the full page section of the PAL, we refer to an edit mode and settings page. If we keep Cancel/Update (or change it to Cancel/Save), this would probably represent an edit mode.

Maybe the stories for this would make more sense as “edit more” and “settings” to mirror the PAL guidance. The PAL site looks like it shows multiple sections of the same page rather than multiple steps like in `CreateFullPage`. Not sure if we’d need multiple steps to complete an edit? We can reach out to Shelley Dignum if we need more clarification.